inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_non_max_suppression.cpp

```diff
@@ -4,39 +4,50 @@
 
 #include "vpu/ngraph/transformations/dynamic_to_static_shape_non_max_suppression.hpp"
 
+#include <vpu/ngraph/operations/static_shape_non_maximum_suppression.hpp>
 #include "vpu/ngraph/operations/dynamic_shape_resolver.hpp"
-#include "vpu/ngraph/operations/dynamic_non_max_suppression.hpp"
 #include "vpu/ngraph/utilities.hpp"
-#include <vpu/utils/error.hpp>
 
+#include <vpu/utils/error.hpp>
 #include "ngraph/graph_util.hpp"
-#include "ngraph/opsets/opset3.hpp"
+#include "ngraph/opsets/opset5.hpp"
 
 #include <memory>
-#include <vpu/ngraph/operations/static_shape_non_maximum_suppression.hpp>
 
 namespace vpu {
 
 void dynamicToStaticNonMaxSuppression(std::shared_ptr<ngraph::Node> node) {
-    auto nms_dynamic = std::dynamic_pointer_cast<ngraph::vpu::op::DynamicNonMaxSuppression>(node);
-    VPU_THROW_UNLESS(nms_dynamic, "dynamicToStaticNonMaxSuppression transformation for {} of type {} expects {} as node for replacement",
-                     node->get_friendly_name(), node->get_type_info(), ngraph::vpu::op::DynamicNonMaxSuppression::type_info);
+    auto nms = std::dynamic_pointer_cast<ngraph::opset5::NonMaxSuppression>(node);
+    VPU_THROW_UNLESS(nms, "dynamicToStaticNonMaxSuppression transformation for {} of type {} expects {} as node for replacement",
+                     node->get_friendly_name(), node->get_type_info(), ngraph::opset5::NonMaxSuppression::type_info);
```

Comment:
Do we have print overload for type_info?

Reply:
Yes.

Reply:
Ok, thanks for explanation!